_sections/act_and_prepare/en/02-do/19-shop-when-at-risk.md

```diff
@@ -0,0 +1,9 @@
+{:.only_toc}
+### Shopping when at risk
+
+{:.do.no_toc}
+### Shopping when at risk
+
+If you can afford to have groceries delivered, or can have someone shop for you, so. Else, if you are elderly or immunocompromised go to the store first thing in the morning when things are less touched.
```

Comment:
"...have someone shop for you, so." should read "...have someone shop for you, do so."

Also, I think substituting "Otherwise" for "Else" would read more naturally (to the non-programmer).